src/main.c

```diff
@@ -707,17 +707,20 @@ static int stop_component(zl_comp_t *comp) {
         comp->name, comp->pid);
 
   pid_t pgid = -comp->pid;
-  if (!kill(pgid, SIGTERM)) {
+  kill(pgid, SIGTERM);
```

Comment:
It's still valuable to check rc of kill(). So would like to keep section of

```
if (!kill()) {
} else {
    DEBUG("kill() failed for %s - %s\n", comp->name, strerror(errno));
    return -1;
}
```

Similar to other kill() usage below.

Reply:
Kept the DEBUG statement for all kill usages